const THROW_EXCEPTION = {
  UNKNOWN_TRANSACTION: 'Something went wrong. <br /> Please try again again!',
  ONCHAIN_TIMEOUT: 'Network too busy. <br /> Please try again!',
};

export { THROW_EXCEPTION };
